docs/about/contributing/templates.md

```diff
@@ -0,0 +1,534 @@
+   coder templates push test-python-template -d .
+   coder create test-workspace --template test-python-template
+   
+   # Format code
+   bun fmt
+   ```
+
+3. **Commit with clear messages**:
+
+   ```bash
+   git add .
+   git commit -m "Add Python development template with FastAPI setup"
+   ```
+
+4. **Open a pull request**:
+   - Use a descriptive title
+   - Explain what the template provides
+   - Include testing instructions
+   - Reference any related issues
+
+## Template examples
+
+### Docker-based template
+
+```terraform
+# Simple Docker template
+resource "docker_container" "workspace" {
+  count = data.coder_workspace.me.start_count
+  image = "ubuntu:20.04"
+  name  = "coder-${data.coder_workspace_owner.me.name}-${data.coder_workspace.me.name}"
+  
+  command = ["sh", "-c", coder_agent.main.init_script]
+  env     = ["CODER_AGENT_TOKEN=${coder_agent.main.token}"]
+}
+```
+
+### AWS EC2 template
+
+```terraform
+# AWS EC2 template
+resource "aws_instance" "workspace" {
+  count         = data.coder_workspace.me.start_count
+  ami           = data.aws_ami.ubuntu.id
+  instance_type = var.instance_type
+  
+  user_data = coder_agent.main.init_script
+  
+  tags = {
+    Name = "coder-${data.coder_workspace_owner.me.name}-${data.coder_workspace.me.name}"
+  }
+}
+```
+
+### Kubernetes template
+
+```terraform
+# Kubernetes template
+resource "kubernetes_pod" "workspace" {
+  count = data.coder_workspace.me.start_count
+  
+  metadata {
+    name = "coder-${data.coder_workspace_owner.me.name}-${data.coder_workspace.me.name}"
+  }
+  
+  spec {
+    container {
+      name  = "workspace"
+      image = "ubuntu:20.04"
+      
+      command = ["sh", "-c", coder_agent.main.init_script]
+      env {
+        name  = "CODER_AGENT_TOKEN"
+        value = coder_agent.main.token
+      }
+    }
+  }
+}
+```
+
+## Common issues and solutions
+
+### Template development
+
+**Issue**: Template fails to create resources
+**Solution**: Check Terraform syntax and provider configuration
+
+**Issue**: Agent doesn't connect
+**Solution**: Verify agent token and network connectivity
+
+**Issue**: Registry modules not working
+**Solution**: Check module versions and variable passing
+
+### Documentation
+
+**Issue**: Template not appearing on registry
+**Solution**: Check frontmatter format and ensure all required fields are present
+
+**Issue**: Icon not displaying
+**Solution**: Verify icon path and file existence
+
+### Platform-specific
+
+**Issue**: Docker containers not starting
+**Solution**: Verify Docker daemon is running and accessible
+
+**Issue**: Cloud resources failing
+**Solution**: Check credentials and permissions
+
+## Get help
+
+- **Examples**: Review real-world examples in the [Coder repository templates](https://github.com/coder/coder/tree/main/examples/templates)
+- **Modules**: Browse available modules at [registry.coder.com/modules](https://registry.coder.com/modules)
+- **Issues**: Open an issue for technical problems
```

Comment:
Link the repo where to create the issue. It should be coder/regsitry and not coder/coder